ascii/distro/gem

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Remove all padding around the art. The art should touch the left and top edges of the file.